core/src/test/java/com/kin/ecosystem/core/data/order/OrderRepositoryTest.java

```diff
@@ -453,7 +454,7 @@ public void onFailure(KinEcosystemException exception) {
 		Thread.sleep(500);
 		ShadowLooper.runUiThreadTasks();
 		verify(blockchainSource, never()).addPaymentObservable(any(Observer.class));
-		verify(remote, never()).submitOrder(anyString(), anyString(), any(Callback.class));
+		verify(remote, never()).submitSpendOrder((String) isNull(), anyString(), any(Callback.class));
```

Comment:
not sure why we are using the ShadowLooper, we should mock the API and just use argument capture and do a response on the callback.